### PR TITLE
Create documentation for psc-package

### PR DESCRIPTION
**Pull request by paf31**
Just some basic notes for now.

### CODE REVIEW

File: psc-package/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+Here is a simple project configuration:
+
+```json
+{
+    "name": "my-project",
+    "set": "psc-0.10.2",
+    "source": "https://github.com/purescript/package-sets.git",
+    "depends": [
+        "prelude"
```
Comment (tmcgilchrist) [inline psc-package/README.md]: @paf31 are the version bounds of the prelude dependency being explicitly dropped here? Or are they taken from the `bower.json` inside `my-project`?

Comment (kRITZCREEK) [inline psc-package/README.md]: They are taken from this file: https://github.com/purescript/package-sets/blob/psc-0.10.2/packages.json

The set gets chosen by the `set` field inside `psc-package.json`.

Comment (tmcgilchrist) [inline psc-package/README.md]: So no loss of version information for a package's dependencies, that's good.

File: psc-package/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    ]
+}
+```
+
+It defines:
+
+- The project name
+- The package set to use to resolve dependencies (this corresponds to a branch or tag of the package set source repository)
```
Comment (paf31) [inline psc-package/README.md]: I'll add something along these lines, thanks.

Comment (paf31) [inline psc-package/README.md]: I've added a note under "Concepts/Package sets"

**Comment by paf31** [comment]
Any other comments on this? If not, I'll merge it.

**Comment by paf31** [comment]
Ok, we can always refine this later.

**Pull request merged**